target/linux/lantiq/image/vr9.mk

```diff
@@ -96,6 +96,36 @@ define Device/arcadyan_vgv7519-nor
 endef
 TARGET_DEVICES += arcadyan_vgv7519-nor
 
+define Device/arcadyan_vgv952cjw33-e-ir
+  $(Device/NAND)
+  SOC := vr9
+  DEVICE_VENDOR := Arcadyan
+  DEVICE_MODEL := VGV952CJW33-E-IR
+  DEVICE_ALT0_VENDOR := Vodafone
+  DEVICE_ALT0_MODEL := Easybox 904xDSL
+  SUPPORTED_DEVICES += VGV952CJW33-E-IR
+  IMAGES := sysupgrade.bin
+  KERNEL_INITRAMFS := kernel-bin | append-dtb | lzma | uImage lzma | pad-offset 16 0 | append-uImage-dummyrootfs
+endef
+
+define Device/arcadyan_vgv952cjw33-e-ir-vpe
+  $(Device/arcadyan_vgv952cjw33-e-ir)
+  DEVICE_VARIANT := VPE
+  DEVICE_ALT0_VARIANT := vpe
```

Comment:
Why lower case for alt title?

Reply:
the below, no idea

Reply:
To me it looks like both should be upper case. Same for SMP DEVICE_VARIANTs below.